tests/src/empty_program.rs

```diff
@@ -0,0 +1,38 @@
+#[cfg(test)]
+mod tests {
+    use crate::utils::{
+        assert_executions_successful, assert_proving_successful, get_env_zkvm_or_default, run_guest,
+    };
+
+    use benchmark_runner::{guest_programs, Action};
+    use ere_dockerized::ErezkVM;
+    use tempfile::tempdir;
+
+    #[tokio::test]
+    async fn execute_empty_program() {
+        let zkvms = get_env_zkvm_or_default(vec![ErezkVM::SP1, ErezkVM::Risc0, ErezkVM::Zisk]);
```

Comment:
This is how we get the zkvm as targets for the run. `get_env_zkvm_or_default` is a helper that reads the `ZKVM` env-var (if any) or fallbacks to the provided default.